apps/web/src/components/app/queue/QueueNextUp.test.tsx

```
@@ -0,0 +1,71 @@
+  QueueItem: ({
+    track,
+    onPlay,
+    onRemove,
+  }: {
+    track: PlayrQueueItem;
+    onPlay: (track: PlayrQueueItem) => void;
+    onRemove: (id: string, e: React.MouseEvent<HTMLButtonElement>) => void;
+  }) => (
+    <div data-testid={`queue-item-${track.uniqueId}`}>
+      {track.title}
+      <button onClick={() => onPlay(track)}>Play</button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)